actors/common/src/main/java/org/sunbird/learner/actors/NotesManagementActor.java

```diff
@@ -335,8 +334,7 @@ private void deleteNote(Request actorMessage) {
       Map<String, Object> req = new HashMap<>();
       req.put(JsonKey.ID, noteId);
       req.put(JsonKey.IS_DELETED, true);
-      String updatedBy = (String) actorMessage.getRequest().get(JsonKey.REQUESTED_BY);
-      req.put(JsonKey.UPDATED_BY, updatedBy);
+      req.put(JsonKey.UPDATED_BY, userId);
```

Comment:
Since the map contains context. It may cause error when updating record. This may need to be removed from map. Test this scenario.